prisma/seed.js

```diff
@@ -0,0 +1,124 @@
+const { PrismaClient } = require('@prisma/client');
+
+const prisma = new PrismaClient();
+
+async function main() {
+  // TODO: perform data seeding dependent on environment
+  await userSeed();
+  await dataSeed();
+}
+
+async function dataSeed() {
+  // TODO: seed some sample data
+  const sizeSettings = ['Small Group', 'Medium Group', 'Large Group', 'Creative Collaboration'];
+  const styleSettings = [
+    'Jazz',
+    'Lyrical',
+    'Ballet',
+    'Open/Fusion',
+    'Modern/Contemporary',
+    'Hip Hop',
+    'Tap',
+    'Cultural',
+    'Musical Theatre',
+    'Live Vocals',
+  ];
+  const levelSettings = ['Easy', 'Intermediate', 'Advanced'];
+  // TODO: add some logging for batch upserts
+  for (let size of sizeSettings) {
+    await prisma.setting.upsert({
+      where: {
+        settings_unique: {
+          type: 'DANCE_SIZE',
+          value: size,
+        },
+      },
+      update: {},
+      create: {
+        type: 'DANCE_SIZE',
+        value: size,
+      },
+    });
+  }
+
+  for (let style of styleSettings) {
+    await prisma.setting.upsert({
+      where: {
+        settings_unique: {
+          type: 'STYLE',
+          value: style,
+        },
+      },
+      update: {},
```

Comment:
Do you need `update: {}`?

Reply:
Yup, it's required field in prisma docs.